backend/app/routes/devices_management_router.py

```diff
@@ -0,0 +1,56 @@
+from http.client import responses
+
+from fastapi import APIRouter, Depends, HTTPException, status
+from fastapi.responses import JSONResponse, Response
+from bson.objectid import ObjectId
+from bson.errors import InvalidId
+
+from app.repository.device_repository import DeviceRepository
+from app.repository.exceptions.repository_exceptions import DatabaseException
+from app.models.device import DeviceModel
+
+
+router = APIRouter(prefix="/devices_management")
+
+
+@router.get("/devices")
+async def get_devices(device_repository: DeviceRepository = Depends(DeviceRepository)) -> JSONResponse:
+    try:
+        response: list = device_repository.find({})
+    except DatabaseException:
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail="Something went wrong with database connection.")
+
+    response = [{**device, "_id": str(device["_id"])} for device in response]
+    return JSONResponse(status_code=status.HTTP_200_OK, content=response)
+
+
+@router.post("/devices")
+async def create_device(
+        new_device: DeviceModel,
+        device_repository: DeviceRepository = Depends(DeviceRepository)
+) -> JSONResponse:
+    try:
+        device_repository.insert(new_device)
+    except DatabaseException:
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail="Something went wrong with database connection.")
+
+    return JSONResponse(status_code=status.HTTP_200_OK, content=new_device.model_dump())
+
+
+@router.delete("/devices/{device_id}")
+async def delete_device(
+        device_id: str,
+        device_repository: DeviceRepository = Depends(DeviceRepository)
+) -> Response:
+    try:
+        device_id = ObjectId(device_id)
+        affected_files: int = device_repository.delete({"_id": device_id})
+    except InvalidId:
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail="device_id has invalid format.")
+    except DatabaseException:
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail="Something went wrong with database connection.")
+
+    if not affected_files:
```

Comment:
`affected_files` to rozumiem liczba usuniętych?
Może `n_affected`?
No i ten if, `if n_affected == 0` czy coś

Reply:
Teraz jeszcze ten komentarz zauważyłem:
- `affected_files` - tak, liczba dokumentów w kolekcji, która w jakiś sposób została zmodyfikowana przy wykonaniu tej operacji (zawsze 0 albo coś dodatniego z tego co pamiętam)
- faktycznie zmieniam to na `n_affected`
- if imo może zostać, bo 0 w pythonie jest falsy value czyli jak będzie `not 0` no to de facto `True` czyli wejdziemy do ifa i rzucimy wyjątek (w tym momencie przetestowane)